src/core/Akka.Remote.Tests/RemoteMessageLocalDeliverySpec.cs

```diff
@@ -86,7 +86,7 @@ public void RemoteActorRefProvider_should_correctly_resolve_valid_LocalActorRef_
                     var ai =
                         Sys.ActorSelection(actorPath).Ask<ActorIdentity>(new Identify(null), TimeSpan.FromSeconds(3)).Result;
 
-                   
+                    Task.Delay(TimeSpan.FromSeconds(3)).Wait(); 
```

Comment:
If this line is not here, the deathwatchnotification will for some reason not be sent from the remote actor